Source Packages/io/rerum/crud/tinyOverwrite.java

```diff
@@ -0,0 +1,167 @@
+        {
+          bodyString.append(line);
+        }
+        requestString = bodyString.toString();
+        try{ 
+            //JSONObject test
+            requestJSON = JSONObject.fromObject(requestString);
+            moveOn = true;
+        }
+        catch(Exception ex){
+            response.setStatus(500);
+            response.getWriter().print(ex);
+        }
+        //If it was JSON
+        if(moveOn){
+            //Get public token for requests from property file
+            String pubTok = manager.getAccessToken();
+            boolean expired = manager.checkTokenExpiry();
+            if(expired){
+                System.out.println("Tiny thing detected an expired token, auto getting and setting a new one...");
+                pubTok = manager.generateNewAccessToken();
+            }
+            //Point to rerum server v1
+            URL postUrl = new URL(Constant.RERUM_API_ADDR + "/overwrite.action");
+            HttpURLConnection connection = (HttpURLConnection) postUrl.openConnection();
+            connection.setDoOutput(true);
+            connection.setDoInput(true);
+            connection.setRequestMethod("PUT");
+            connection.setUseCaches(false);
+            connection.setInstanceFollowRedirects(true);
+            connection.setRequestProperty("Content-Type", "application/json; charset=utf-8");
+            connection.setRequestProperty("Authorization", "Bearer "+pubTok);
+            connection.connect();
+            try{
+                DataOutputStream out = new DataOutputStream(connection.getOutputStream());
+                //Pass in the user provided JSON for the body of the rerumserver v1 request
+                out.writeBytes(requestJSON.toString());
+                out.flush();
+                out.close(); 
+                codeOverwrite = connection.getResponseCode();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream(),"utf-8"));
+                while ((line = reader.readLine()) != null){
+                    //Gather rerum server v1 response
+                    sb.append(line);
+                }
+                reader.close();
+            }
+            catch(IOException ex){
+                //Need to get the response RERUM sent back.
+                BufferedReader error = new BufferedReader(new InputStreamReader(connection.getErrorStream(),"utf-8"));
+                String errorLine = "";
+                while ((errorLine = error.readLine()) != null){  
+                    sb.append(errorLine);
+                } 
+                error.close();
+            }
+            connection.disconnect();
+            response.setStatus(codeOverwrite);
+            response.getWriter().print(sb.toString());
+        }
+        
+    }
+
+    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
+    /**
```

Comment:
There is a PR that says we shouldn't support incorrect methods, maybe get rid the doGet here and let a GET /overwrite fail hard.